ui/src/components/TestRunnerFields/HomeDomainField.tsx

```diff
@@ -86,11 +94,20 @@ export const HomeDomainField = ({
   };
   return (
     <>
-      <Input
-        id="homeDomain"
-        label="Home Domain"
-        onChange={(e) => handleHomeDomainChange(e.target.value)}
-      />
+      <FieldWrapper>
+        <Input
+          id="homeDomain"
+          label="Home Domain"
+          onChange={(e) => handleHomeDomainChange(e.target.value)}
+        />
+        <TooltipInfoButton>
+          <>
+          <p>
+            Input the domain that hosts your anchor's <TextLink underline={true} href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md">Stellar Info File</TextLink>.
```

Comment:
same here - `’` for `'`

Reply:
Also, you can just do `<TextLink underline href=...`, instead of explicitly setting underline to `true`